pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -1061,9 +1061,15 @@ func (o *MustGatherOptions) newPod(node, image string, hasMaster bool) *corev1.P
 // BackupGathering is called if the full must-gather has an error.  This is useful for making sure we get *something*
 // no matter what has failed.  It should be focused on universal openshift failures.
 func (o *MustGatherOptions) BackupGathering(ctx context.Context, errs []error) {
+	targets := []string{
+		"clusterversion.v1.config.openshift.io",
+		"clusteroperators.v1.config.openshift.io",
+		"namespace/openshift-cluster-version",
```

Comment:
We are trying to be bare minimum in here. I don't think, we would want to initiate another inspection for all namespaces in this fall back mechanism.

Reply:
> We are trying to be bare minimum in here. I don't think, we would want to initiate another inspection for all resources in a namespace in this fall back mechanism.

When the upgrade fails and the must-gather fails (this happened yesterday), we end up with a resulting backup collection that lacks the cluster-version logs, so we cannot diagnose why.  The namespace collection of this individual namespace will provide that information.